charts/hlf-k8s/values.yaml

```diff
@@ -300,6 +300,8 @@ enrollments:
   csrHost: service-hostname
 
 hooks:
+  # Service account used for the hooks, must not be part of the release and have the rights to delete secrets
+  serviceAccount: tiller
```

Comment:
I don't think we'd want to reference tiller at all in the new chart. 

What would you think of adding a new template with `kind: ServiceAccount` that creates the SA required by the post-hook?